bin/mail/2016-03-21-teacher-recruitment/generate-inactive-teachers

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env ruby
+require_relative '../mailing-common/mailing-list-utils'
+require 'set'
+
+# Inactive Code Studio teacher accounts (not teaching a course, or haven't added students)	US only
+
+US_TEACHERS = Set.new
+SOLR_PARAMS = {
+  q: 'kind_s:user && role_s:teacher',
+  fq: 'create_ip_country_s:"United States"',
+  fl: 'email_s',
+  rows: 10000
```

Comment:
The rows param only controls how many results are returned in a given page, and the default rows is 10. The .each method handles paging. 

I just ran this and got:
```
168717 US teachers loaded from solr
```

This is following the [existing pattern](https://github.com/code-dot-org/code-dot-org/blob/345ce81fa9f21378e49cfc4136e23a41cc195488/bin/mail/mailing-common/mailing-list-utils.rb#L61)